docstore/awsdynamodb/create_tables.sh

```diff
@@ -42,6 +42,7 @@ aws dynamodb create-table \
         AttributeName=Player,AttributeType=S \
         AttributeName=Score,AttributeType=N \
         AttributeName=Time,AttributeType=S \
+        AttributeName=WithGlitch,AttributeType=BOOL \
```

Comment:
This doesn't work, the only possible values are S, N, B. I am not sure what to use for a BOOL value....

Although, the test seems to pass without this so maybe it's not necessary?

Reply:
Yes you are correct, I have removed it